README.md

```diff
@@ -56,10 +56,29 @@ A project to automate expunging qualifying criminal records.  This project is do
 	package manager which also automatically creates and manages virtual 
 	environments.
 
+4. Install additional libraries needed for running the backend natively or with pipenv:
+
+    * **Mac**
+
+      ```
+      brew install postgres
```

Comment:
Brew command for postgres is actually `brew install postgresql` <- this is only needed if the user doesn't have postgres installed already. More information regarding the install can be found here https://wiki.postgresql.org/wiki/Homebrew

Reply:
I just want to point out that for Windows users who want to use Ubuntu there is an Ubuntu app on the Microsoft Store that runs Ubuntu natively in Windows, no need to partition your disk and install yourself.

Installing Ubuntu app:
https://docs.microsoft.com/en-us/windows/wsl/install-win10

Installing Docker Desktop on Windows:
https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly

It does take some PowerShell-fu that was new to me coming from Linux but once Ubuntu and Docker Desktop are configured, you can run the Ubuntu app from the Windows menu or type 'bash' from PowerShell to get a bash prompt (or use a bash-friendly console like ConEmu)

Reply:
Woops I will fix that line add that link for more information re the homebrew install. 

Given @erikjasso 's comment, should we just point that out at the top of the README and remove instructions for installing components on Windows?